types/office-js/index.d.ts

```diff
@@ -29867,27 +30124,35 @@ declare namespace Excel {
      * Represents the value of a cell containing a number with a format string. Number format strings must conform to Excel guidelines. To learn more, see {@link https://support.microsoft.com/office/c0a1d1fa-d3f4-4018-96b7-9c9354dd99f5  | Review guidelines for customizing a number format}.
      * In this scenario, the format is applied to the value and not to the cell, so the value retains its format string throughout calculation.
      *
+     * @deprecated since [Api set: ExcelApi 1.19]. Use {@link https://learn.microsoft.com/javascript/api/excel/excel.doublecellvalue | DoubleCellValue} instead.
```

Comment:
I'm concerned about what the docs user experience will be like.

For example, [Office.MeetingSuggestion](https://learn.microsoft.com/en-us/javascript/api/outlook/office.meetingsuggestion?view=outlook-js-preview) says `@deprecated Use {@link https://learn.microsoft.com/office/dev/add-ins/outlook/contextual-outlook-add-ins | regular expression rules} instead.` and looks like:

```
Warning

This API is now deprecated.

Use regular expression rules instead.
```

My concern is that this will look like:

```
Warning

This API is now deprecated.

since Api set: ExcelApi 1.19. Use DoubleCellValue instead.
```

Consider saying something like the following instead.

```suggestion
     * @deprecated As of [Api set: ExcelApi 1.19], use {@link https://learn.microsoft.com/javascript/api/excel/excel.doublecellvalue | DoubleCellValue} instead.
```

Then we should end up with something like:

```
Warning

This API is now deprecated.

As of Api set: ExcelApi 1.19, use DoubleCellValue instead.
```

Reply:
Here and other instances